test/behaviour/BUILD

```diff
@@ -1,16 +1,18 @@
 #
-# Copyright (C) 2020 Grakn Labs
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
That would be nice, yes. But I don't know how we could do that? 🤔

Reply:
> > I think there's a strong case now to keep the UX to be: Grakn.Client client = new GraknClient(...); where the user also has to import grakn.core.client.rpc.GraknClient.
> 
> This should be fine provided we do actually rename `RPCClient` to `GraknClient`.
> 
> This would make the class name inconsistent with the rest of the package, but I think that's fine.

done